Resources/public/templates/partials/tree.hbt

```diff
@@ -1,6 +1,6 @@
 <ul class="ez-tree-level">
 {{#each children}}
-    <li class="ez-tree-node{{#if state.leaf}} is-tree-node-leaf{{/if}} {{#if state.open}}is-tree-node-open{{else}}is-tree-node-close{{/if}}" data-node-id="{{id}}">
+    <li class="ez-tree-node{{#if state.leaf}} is-tree-node-leaf{{/if}} {{#if state.open}}is-tree-node-open{{else}}is-tree-node-close{{/if}}" data-node-id="{{id}}" title="{{ data.contentInfo.name }}">
```

Comment:
Hmmm. Interesting.

I mostly agree with @joaoinacio's analysis. But can't we keep the name as a scenario side piece of information ? If whenever we create a location for a scenario, we need to reference its name, we can map the name to the id.

Except for built-in locations that we would use, we can always do it, right ?

It would avoid having to add the name as an attribute.